singularity/Singularity

```diff
@@ -47,7 +44,8 @@ From: condaforge/mambaforge
     cd /opt/cta
 
     # Install nectarchain
-    mamba env create --quiet --file /tmp/environment.yml --prefix /opt/conda/envs/nectarchain
+    git clone https://github.com/cta-observatory/nectarchain.git
```

Comment:
Cloning inside the singularity file means that you always build the default branch. So your CI tests wouldn't actually catch the case where a PR breaks the singularity build. 

I think you should add the local files.

Reply:
You're absolutely right, thanks a lot! That was actually what I also intended to do at the beginning, but I forgot along the way that everything needed is already there thanks to the checkout action :sweat_smile:.

Reply:
@maxnoe , should be properly done now